base/include/SoundRecord.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "Module.h"
+
+class SoundRecordProps : public ModuleProps
+{
+public:
+    SoundRecordProps(
+        int _sampleRate,
+        int _channel,
+        int _device,
+        int _proccessingRate) : sampleRate(_sampleRate),
+                                channel(_channel),
+                                device(_device),
+                                proccessingRate(_proccessingRate)
+    {
+    }
+    int sampleRate;
+    int channel;
+    int device;
+    int proccessingRate;
+};
+
+class SoundRecord : public Module
```

Comment:
The name SoundRecord is quiet confusing.. An intuitive name like AudioCaptureSrc should be used. as proposed in #64